CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
-= Changes for the _Jenkins Parameterized Scheduler Plugin_
+## Changes for the _Parameterized Scheduler Plugin_
 
-* 0.1 : First release to the world
-* 0.2 : Not user facing change
-* 0.4-beta-1 Pipeline support Beta [JENKINS-37139](https://issues.jenkins-ci.org/browse/JENKINS-37139)
-* 0.4 Pipeline support
-* 0.5 Added [Declarative pipeline syntax support](https://github.com/jenkinsci/parameterized-scheduler-plugin#declarative-pipeline-configuration-example) and [JENKINS-44031](https://issues.jenkins-ci.org/browse/JENKINS-44031) resolved.
+0.1. First release to the world
```

Comment:
Not sure what you are trying to do here? Those should be bullet points, why do you want to transform these into paragraphs?

Reply:
Or make them `###` possibly, one entry per version?

Reply:
@colourmeamused friendly ping ^